docs/artifacts/lockfile-blind-spot.html

```
@@ -0,0 +1,370 @@
+<figure>
+<pre class="mermaid">
+%%{init:{'theme':'base','themeVariables':{'background':'#FFFFFF','primaryColor':'#FFFFFF','primaryTextColor':'#16202B','primaryBorderColor':'#C3CCD6','lineColor':'#5A6875','fontFamily':'Karla, system-ui, sans-serif','fontSize':'14px'}}}%%
+flowchart LR
+  BUMP["a pin bump"]
+
+  MT["mise.toml<br/><i>the pin — always</i>"]
+  ML["mise.lock<br/><i>per-platform URL + checksum<br/>always, while lockfile = true</i>"]
+  MAN["sources/*.manifest<br/><i>upstream tag + SHA<br/>if the tool has one</i>"]
+  CUR["currency.toml<br/><i>expected version<br/>only if hardcoded — 5 blocks</i>"]
+
+  BUMP -->|"always"| MT
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)